crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -80,6 +80,15 @@ def maintenance_updates_installed?
       end
     end
 
+    def clusters_healthy?
+      service_object = CrowbarService.new(Rails.logger)
+      cluster_health = service_object.check_cluster_health
+      unless cluster_health.empty?
+        Rails.logger.warn("clusters report some problems: #{cluster_health}")
```

Comment:
It's a hash describing various types of errors. Is anything wrong with logging that?